content/blog/2023/09/kwctl-sha.md

```diff
@@ -0,0 +1,35 @@
+---
+title: kwctl SHA support
+authors:
+  - Fabrizio Sestito
+date: 2023-09-06
+---
+
+Lately, we have been focusing on improving the developer experience of Kubewarden, and in general,
+we are striving to implement features that are being requested by the community.
+
+# Reference policies by their SHA
+
+Today we are happy to announce that we have released a new version of `kwctl` that supports
```

Comment:
I think we should change this sentence; the publish date of the post will not match the release of 1.7.0, and as it is worded, it feels we did a new release. I understand that the "today" is for only "being happy to announce" and not "we have released", but it's ambiguous.

Maybe something like:
> This effort got rewarded, and since the `v1.7.0` release `kwctl` now supports referencing policies by their SHA.